translations/handleAndroidTranslations.sh

```diff
@@ -20,6 +20,9 @@ tx push -s
 # pull translations
 tx pull -f -a --minimum-perc=50
 
+# copy transifex strings to fastlane
+app/scripts/metadata/generate_metadata.py
```

Comment:
Why add it after the push? I would add it before the push is done. In this case the metadata is updated one day earlier on transifex and does not need to wait for two transifex sync runs.

Reply:
It is after pull and before push?
L21: tx pull
L28: commit (but where is the push?)

Reply:
> L28: commit & push

That is the push to github. I meant the push to transifex (L18). Just place it between 15 and 17. 😉